.buildkite/heartbeat/scripts/unit-tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source .buildkite/env-scripts/linux-env.sh
+
+echo "--- Running Unit Tests"
+sudo chmod -R go-w heartbeat/
```

Comment:
leave a comment as mentioned above about removing it after we've switched to custom images.